frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -46,6 +74,22 @@ interface Props {
   max?: number;
 }
 
+/**
+ * @typedef ContentProps
```

Comment:
What's the relationship, if any, between `Props` and `ContentProps`? It looks like they have almost the exact same fields, except for the optional `disableSetNewButton` in `Props`?

Reply:
`ContentProps` was updated to use in Multistep popup. `disableSetNewButton` is to show/hide `Set New` button in `ValueModal` which is deprecated as we do not use it anywhere on our existing UI flow